netci.groovy

```diff
@@ -221,34 +231,45 @@ def CreateStyleCheckTasks = { taskString, taskName, checkName ->
 // INNER LOOP TASKS
 // ----------------
 
-CreateBuildTasks('Windows_NT', null, null, "-winBlue", true, null, null)
+CreateBuildTasks(latestWindowsMachine, latestWindowsMachineTag, null, "-winBlue", true, null, null)
```

Comment:
Ah; you removed an additional null that you didn't need to. Needs a `null` after `latestWindowsMachineTag`

Reply:
Ah, good catch, thanks.